modules/contact-form/css/gutenblock.scss

```diff
@@ -0,0 +1,84 @@
+
+
+.jetpack-field-label.jetpack-field-label.jetpack-field-label {
+	border-color: #fff;
+	font-weight: bold;
+	margin: 0;
+	margin-bottom: 2px;
+	padding-left: 1px;
+	width: calc( 100% - 100px );
+}
+
+.jetpack-field-label.jetpack-field-label.jetpack-field-label:focus,
+.jetpack-field-label.jetpack-field-label.jetpack-field-label:hover {
+	border-color: #8d96a0;
```

Comment:
We don't have a mechanism to use Gutenberg colour variables directly yet, but let's define them in this file and use colours via variables like we do all across in Calypso.

See example: https://github.com/Automattic/wp-calypso/blob/a5c963f19173773d1732b3f23e2a60b961faa3a2/client/gutenberg/extensions/markdown/editor.scss